### PR TITLE
Change GitHub Actions to dedicated server

### PR DESCRIPTION
**Pull request by TomPohys**
Change Github Actions to the dedicated server name "ralph"

### CODE REVIEW

**Comment by nyurik** [comment]
Also, note that the test did not complete -- so the custom server must still be down

**Comment by daliborjanak** [comment]
It is, we are working on it. It will be solved today

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ jobs:
           MIN_ZOOM: 0
           MAX_ZOOM: 14
         run: |
+          # For now, change the quickstart values directly in the .env file
```
Comment (nyurik) [inline .github/workflows/tests.yml:26]: why is this being added?

Comment (TomPohys) [reply .github/workflows/tests.yml:26]: MAX_ZOOM: 14 was not taken. Please look into old integrity test on monaco-dc_config.yaml

**Pull request merged**